core/src/main/java/com/orientechnologies/orient/core/storage/cache/pages/OPassthroughPageCache.java

```diff
@@ -57,7 +57,7 @@ public void releasePage(OCacheEntry cacheEntry, OWriteCache writeCache) {
 
   @Override
   public void releaseFilePages(long fileId, OWriteCache writeCache) {
-    // do nothing
+    releasePage(fileId, writeCache);
```

Comment:
Passthrough cache caches nothing, so there is nothing to release from it. I doubt this even compiles, since there is no `releasePage` method accepting `long` as a first argument. "do nothing"-like comment means that the method is intentionally left blank.